.changes/process-std.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effection/process": "minor"
+---
+
+The `stdout` and `stderr` properties now return IoStreams instead of regular streams. Addiotionally, the `buffered` option has been removed.
```

Comment:
```suggestion
The `stdout` and `stderr` properties now return `OutputStream`s instead of regular streams. Additionally, the `buffered` option has been removed.
```